<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreProcessorGridAttributeAbstractCellDataItem_PropertyDialog</class>
 <widget class="QDialog" name="PreProcessorGridAttributeAbstractCellDataItem_PropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grid Cell Attribute Display Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
   <item>
    <widget class="QGroupBox" name="colormapGroupBox">
     <property name="title">
      <string>Colormap (This setting is common with Geographic Data)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidgetContainer" name="widgetContainer" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,1">
     <item>
      <widget class="TransparencyWidget" name="transparencyWidget" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="lineWidthLabel">
       <property name="text">
        <string>Line Width:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="lineWidthSpinBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>&amp;Import...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>&amp;Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWidgetContainer</class>
   <extends>QWidget</extends>
   <header>misc/qwidgetcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransparencyWidget</class>
   <extends>QWidget</extends>
   <header>guibase/widget/transparencywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreProcessorGridAttributeAbstractCellDataItem_PropertyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreProcessorGridAttributeAbstractCellDataItem_PropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
